### ISSUE TITLE
cURL import with '$' in request body are malformed

### ISSUE THREAD

**Issue by AnkitMasand** [a85/POSTMan-Chrome-Extension#5737]
I tried importing a sample Curl request from my local machine to the postman app. The import succeeded but some fields in the request-body that had "$" at the beginning of the name, transformed into a blank space. 

1. Go to postman app
2. Click on import -> paste raw text
3. Paste a sample curl request with a request-body. Eg: {details : {$id :1, name:something }}
4. After the import is successful, the generated request-body looks something like :
{details : {"" :1, name:something }}

App information 
 - App Type [Native App]
 - Postman Version [6.6.1]
 - OS: [macOS High Sierra 10.13.4]

**Comment by numaanashraf** [comment]
@AnkitMasand Please provide a sample cURL snippet we can use to replicate this issue.

**Comment by AnkitMasand** [comment]
Sure. Attaching a sample code.

`curl --location --request POST "https://sample.com" \
  --header "Content-Type: application/json" \
  --data "{
  \"details\": {
    \"$id\": \"11\",
    \"sample\": {
      \"name\": \"ankit\"
    }
  }
}"`

**Comment by abhijitkane** [comment]
The arg parsing library automatically resolves `$id` as an environment variable. We'll need to figure out how to block this.

**Comment by abhijitkane** [comment]
@AnkitMasand Can you confirm this fixed in the latest Postman Canary (https://getpostman.com/canary)?

You'll need to remove the line-ending backslashes:
```
curl --location --request POST "https://sample.com" --header "Content-Type: application/json" --data "{ \"details\": { \"$id\": \"11\", \"sample\": { \"name\": \"ankit\" } } }"
```

**Comment by AnkitMasand** [comment]
curl --location --request POST "https://sample.com" --header "Content-Type: application/json" --data "{ "details": { "$id": "11", "sample": { "name": "ankit" } } }"

@abhijitkane I executed the above Curl command. Attaching a screenshot from the latest version of the Canary app ( Version 7.1.1-canary06 (7.1.1-canary06) ). Doesn't seem like it has been fixed.

![image](https://user-images.githubusercontent.com/21065626/59009652-1ef86c80-884c-11e9-84ad-c3ea18163765.png)
